.github/workflows/nightly.yml

```
@@ -28,29 +28,40 @@ jobs:
+        mypy FollowWeb_Visualizor
+        ruff format --check FollowWeb_Visualizor tests
     
     - name: Test package build
-      run: python tasks.py build
+      run: |
+        cd FollowWeb
+        python -m build
     
     - name: Upload coverage reports
       if: matrix.os == 'ubuntu-latest' && matrix.python-version == '3.11'
       uses: codecov/codecov-action@v3
```

actionlint (1.7.8)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)